docs/TuxOwnerClass.md

```diff
@@ -0,0 +1,318 @@
+
+NOTE: `nearestOwnerProps` should be treated as immutable just like standard component props. Thus, you should never modify the `nearestOwnerProps` object. It is designed to store static methods and data.
+
+```
+    ownerClassProps.registerOwnerProps = function () {
+      //'this' context will be the owner component
+      return {
+        createRoom: this.createRoom.bind(this),
+        deleteRoom: this.deleteRoom.bind(this),
+        updateRoom: this.updateRoom.bind(this)
+      };
+    };
+```
+
+Now, children of the `OwnerClass` can access these props via:
+
+```
+    this.nearestOwnerProps.createRoom;
+    this.nearestOwnerProps.deleteRoom;
+    this.nearestOwnerProps.updateRoom;
+```
+
+***
+
+#### <a id="createOwnerClass-OwnerClass"></a>Return - `OwnerClass` - type: OBJECT [#](#createOwnerClass-OwnerClass)
+The `createOwnerClass` method returns a `TuxOwnerClass` which is an extension on top of a React Class. You can then simply use it in render methods as you would any other React Class.
+
+***
+
+### <a id="TuxOwnerClass-Complete-Example"></a>4) TuxOwnerClass Complete Example [#](#TuxOwnerClass-Example)
+Taking all that we have learned so far let's take a look at a more realistic example.
+
+```
+    var React = require('tux/React');
+    var roomStore = require('../stores/RoomStore');  //room store that we will get data from
+    var roomActions = require('../actions/RoomActions'); //room actions that we will dispatch
+    var RoomCreateForm = require('./RoomCreateForm');
+    var Rooms = require('./Rooms');
+
+    var RoomViewOwner = React.createOwnerClass({
+      getInitialState: function () {
+        return {
+          rooms: roomStore.getAll()
+        };
+      },
+
+      componentDidMount: function () {
+        roomActions.get();
+      },
+
+      connectOwnerToStore: function () {
+        return {
+          store: roomStore,
+          listener: function () {
+            this.setState({
+              rooms: roomStore.getAll()
+            });
+          }.bind(this)
+        };
+      },
+
+      registerOwnerProps: function () {
+        return {
+          handleCreate: roomActions.create,
+          handleUpdate: roomActions.update,
+          handleDelete: roomActions.delete
+        };
+      },
+
+      render: function () {
+        return (
+          <div>
+            <RoomCreateForm />
+            <Rooms rooms={this.state.rooms} />
+          </div>
+        );
+      }
+    });
+
+    React.render(<RoomViewOwner />, document.getElementById("main"));
+```
+
+Hopefully this shows some of the power of the `OwnerClass`. Firstly, we would normally have to do the work of adding and removing listeners to/from our store within the proper lifecycle methods. Thanks to the `connectOwnerToStore` that is no longer necessary.
```

Comment:
Would it be good to mention that connectOwnerToStore adds and removes the listeners from the proper life cycle methods?